jarbas/core/fields.py

```diff
@@ -0,0 +1,29 @@
+from datetime import datetime
+
+from rows.fields import DateField
+from rows.fields import IntegerField as RowsIntegerField
+
+
+class IntegerField(RowsIntegerField):
+
+    @classmethod
+    def deserialize(cls, value, *args, **kwargs):
+        try:
+            # Rows cannot convert values such as '2011.0' to integer
+            value = float(value)
```

Comment:
Replace with `value = int(float(value))` so the method will always return an `int` object (or `None`).